packages/@react-stately/button/src/ButtonGroupCollection.ts

```diff
@@ -17,9 +17,13 @@ export class ButtonGroupCollection<T extends ReactElement> implements Collection
   public items: T[];
   private keys: Key[];
 
-  constructor(items: T[]) {
+  constructor(items: T[], flipKeyOrder: boolean) {
     this.items = items;
     this.keys = items.filter(item => !item.props.isDisabled).map(child => child.key);
+    
+    if (flipKeyOrder) {
```

Comment:
interesting, why fix this here instead of over in the keyboard delegate? https://github.com/adobe-private/react-spectrum-v3/blob/master/packages/%40react-stately/button/src/ButtonGroupKeyboardDelegate.ts

Reply:
Talked with @LFDanLu, he's been flip flopping on this. Here is roughly what it looks like to do it in the delegate
https://github.com/adobe-private/react-spectrum-v3/pull/223/commits/fa883eb5aac1d202d77bbda5f3531c8ce690ce19#diff-91caa43fc876d908a04566d2db6b7b6eL19

He'd like more opinions on this

my opinion is that if we do it in the delegate, we don't need to reverse the keys and it should keep our state separated from the actual keys so that we can swap them out as needed

Reply:
I agree with doing it in the delegate. Rather than a `flipHorizontal` boolean, I'd pass the direction directly though.

Reply:
Sounds good, will change it back so the logic is in the delegate. As for the `flipHorizontal`, it exists since I need to check the `direction` and the ButtonGroup's `orientation`.  We only want to flip the getKeyBefore/After logic if it is a horizontal button group, vertical button groups should be unaffected.